Sources/Datadog/Logger.swift

```diff
@@ -17,6 +17,12 @@ public enum LogLevel: Int, Codable {
     case critical
 }
 
+public enum ErrorAttributes {
```

Comment:
If we want to keep them as part of the public API:
- [ ] `LogAttributes` (as its [Android counterpart](https://github.com/DataDog/dd-sdk-android/blob/master/dd-sdk-android/src/main/kotlin/com/datadog/android/log/LogAttributes.kt)) is a better name as it scopes those attributes - it will be more obvious that those attributes can be used with Logs, but not RUM. It will also provide better API discoverability.
- [ ] It needs public API comment.

Those are only few of all the standard attributes that we support. IMO, instead of listing just a few, we should either list all of them or none (so, make this `internal`). Currently, it may be misleading to the users who found the list of standard attributes on our website, but only few of them listed in public API.

Reply:
i make them `internal` to stay in the scope of this PR, we can add all the attributes as public in another PR later